sdn_tests/pins_ondatra/pins_deps.bzl

```diff
@@ -0,0 +1,211 @@
+                                              )""",
+        )
+    if not native.existing_rule("com_jsoncpp"):
+        http_archive(
+            name = "com_jsoncpp",
+            url = "https://github.com/open-source-parsers/jsoncpp/archive/1.9.4.zip",
+            strip_prefix = "jsoncpp-1.9.4",
+            build_file = "@//:bazel/BUILD.jsoncpp.bazel",
+            sha256 = "6da6cdc026fe042599d9fce7b06ff2c128e8dd6b8b751fca91eb022bce310880",
+        )
+    if not native.existing_rule("com_github_ivmai_cudd"):
+        http_archive(
+            name = "com_github_ivmai_cudd",
+            build_file = "@//:bazel/BUILD.cudd.bazel",
+            strip_prefix = "cudd-cudd-3.0.0",
+            sha256 = "5fe145041c594689e6e7cf4cd623d5f2b7c36261708be8c9a72aed72cf67acce",
+            urls = ["https://github.com/ivmai/cudd/archive/cudd-3.0.0.tar.gz"],
+        )
+    if not native.existing_rule("com_gnu_gmp"):
+        http_archive(
+            name = "com_gnu_gmp",
+            urls = [
+                "https://gmplib.org/download/gmp/gmp-6.2.1.tar.xz",
+                "https://ftp.gnu.org/gnu/gmp/gmp-6.2.1.tar.xz",
+            ],
+            strip_prefix = "gmp-6.2.1",
+            sha256 = "fd4829912cddd12f84181c3451cc752be224643e87fac497b69edddadc49b4f2",
+            build_file = "@//:bazel/BUILD.gmp.bazel",
+        )
+    if not native.existing_rule("com_github_z3prover_z3"):
+        http_archive(
+            name = "com_github_z3prover_z3",
+            url = "https://github.com/Z3Prover/z3/archive/z3-4.8.12.tar.gz",
+            strip_prefix = "z3-z3-4.8.12",
+            sha256 = "e3aaefde68b839299cbc988178529535e66048398f7d083b40c69fe0da55f8b7",
+            build_file = "@//:bazel/BUILD.z3.bazel",
+        )
+    if not native.existing_rule("rules_foreign_cc"):
+        http_archive(
+            name = "rules_foreign_cc",
+            sha256 = "d54742ffbdc6924f222d2179f0e10e911c5c659c4ae74158e9fe827aad862ac6",
+            strip_prefix = "rules_foreign_cc-0.2.0",
+            url = "https://github.com/bazelbuild/rules_foreign_cc/archive/0.2.0.tar.gz",
+        )
+    if not native.existing_rule("rules_proto"):
+        http_archive(
+            name = "rules_proto",
+            urls = [
+                "https://github.com/bazelbuild/rules_proto/archive/3f1ab99b718e3e7dd86ebdc49c580aa6a126b1cd.tar.gz",
+            ],
+            strip_prefix = "rules_proto-3f1ab99b718e3e7dd86ebdc49c580aa6a126b1cd",
+            sha256 = "c9cc7f7be05e50ecd64f2b0dc2b9fd6eeb182c9cc55daf87014d605c31548818",
+        )
+    if not native.existing_rule("sonic_swss_common"):
+        # We use `git_repository` over `http_archive` only because this is a private repo
+        # requiring SSH authentication.
+        git_repository(
+            name = "sonic_swss_common",
+            commit = "672b1cfe1914489b79a91fff8a539b6c195f8959",
+            remote = "git@github.com:pins/sonic-swss-common.git",
```

Comment:
@parth-opensrc What would be the sonic-net equivalent commit id ? We can't use PINS refpt here.

Reply:
I think this repo may not be needed for ondatra, so can be removed for now.
I will clean up this file in future.

Reply:
@VSuryaprasad-HCL Can you remove this and re-attach the pass build logs.

Reply:
I have addressed the changes and reattached the build log.